oidc-ui/Dockerfile

```
@@ -0,0 +1,133 @@
+ENV plugins_format=iife
+
+# set working directory for the user
+WORKDIR /home/${container_user}
+
+# install packages and create user
+RUN groupadd -g ${container_user_gid} ${container_user_group} \
+    && useradd -u ${container_user_uid} -g ${container_user_group} -s /bin/sh -m ${container_user} \
+    && mkdir -p /var/run/nginx /var/tmp/nginx ${i18n_path} ${theme_path} ${image_path} ${plugins_path} ${plugins_path}/temp \
+    && chown -R ${container_user}:${container_user} /usr/share/nginx /var/run/nginx /var/tmp/nginx ${i18n_path} ${theme_path} ${image_path} ${plugins_path} ${plugins_path}/temp
+
+ADD configure_start.sh configure_start.sh
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)